internal/testutil/testutil.go

```diff
@@ -116,3 +118,10 @@ func createExclusionsList(exclusionsText string, t testing.TB) []portpair {
 	}
 	return exclusions
 }
+
+// This prevent catching the leak generated by opencensus defaultWorker.Start which at
+// this time is part of the package's init call.
+// See https://github.com/open-telemetry/opentelemetry-collector/issues/9165#issuecomment-1874836336 for more context.
+func IgnoreOpenCensusWorkerLeak() goleak.Option {
```

Comment:
Not worth all the complex circular dependencies for a 1 line func. Would suggest to inline the function.